packages/frontend/core/src/modules/quicksearch/impls/docs.ts

```diff
@@ -57,23 +60,33 @@ export class DocsQuickSearchSession
       } else {
         const resolvedDoc = resolveLinkToDoc(query);
```

Comment:
Can we separate this part from `DocsQuickSearchSession` and implement a separate `DocLinkQuickSearchSession`?

Reply:
Good idea. This is easier to maintain.